src/domain/accounts/services/src/services/auth_policy_service.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::sync::Arc;
+
+use kamu_accounts::CurrentAccountSubject;
+use thiserror::Error;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#[async_trait::async_trait]
+pub trait AuthPolicyService: Sync + Send {
+    fn check(&self) -> Result<(), AuthPolicyCheckError>;
+}
```

Comment:
The abstract trait should leave in "domain" part of the bounded context, not in the implementation, which are "services".